examples/gnrc_border_router/Makefile

```diff
@@ -62,6 +62,20 @@ else ifeq (uhcp,$(PREFIX_CONF))
   USEMODULE += gnrc_uhcpc
 else ifeq (auto_subnets,$(PREFIX_CONF))
   USEMODULE += gnrc_ipv6_auto_subnets_simple
+else ifeq (static,$(PREFIX_CONF))
+  IPV6_ADDR ?= 2001:db8:1::1
+  IPV6_DEFAULT_ROUTER ?= fe80::1
+
+  ifndef CONFIG_GNRC_IPV6_STATIC_ADDR
+    CFLAGS += -DCONFIG_GNRC_IPV6_STATIC_ADDR=\"$(IPV6_ADDR)\"
+  endif
+  ifndef CONFIG_GNRC_IPV6_STATIC_PREFIX
+    CFLAGS += -DCONFIG_GNRC_IPV6_STATIC_PREFIX=\"$(IPV6_PREFIX)\"
+  endif
+  ifndef CONFIG_GNRC_IPV6_STATIC_DEFAULT_ROUTER
+    CFLAGS += -DCONFIG_GNRC_IPV6_STATIC_DEFAULT_ROUTER=\"$(IPV6_DEFAULT_ROUTER)\"
+  endif
+  USEMODULE += gnrc_ipv6_static_addr
```

Comment:
As the Kconfig is evaluated by the build system, this needs to come after the

```
include $(RIOTBASE)/Makefile.include
```

down below (where the dhcpv6 config is as well)

Reply:
Minus the `USEMODULE += gnrc_ipv6_static_addr` of course.. overshot my marking a bit, sorry.